config/service_providers.yml

```diff
@@ -161,6 +161,38 @@ production:
     attribute_bundle:
       - email
 
+# Micro-purchase
+  'urn:gov:gsa:saml:2.0.profiles:sp:sso:micropurchase-staging':
+    acs_url: 'https://micropurchase-staging.18f.gov/auth/saml/callback'
+    assertion_consumer_logout_service_url: 'https://micropurchase-staging.gov/auth/saml/logout'
+    sp_initiated_login_url: 'https://micropurchase-staging.18f.gov/login'
+    block_encryption: 'aes256-cbc'
+    cert: 'sp_micropurchase'
+    agency: 'TTS Acquisition'
+    friendly_name: 'Micro-purchase Staging'
+    logo: 'generic.svg'
```

Comment:
nah, I don't think we need it. Esp since it's text rather than an img last I checked